### PR TITLE
Add camperbot rooms and close #148

### LINKED ISSUE #148: Add gitterbot to remaining FCC rooms
Gitterbot is awesome. Campers are using it like crazy. Have you been monitoring the statistics?
He does a few key things remarkably well
I was wondering whether you could roll him out in the rest of our rooms
There are about 400 of these - is there a good way to do these in batch? They’re listed here: https://github.com/FreeCodeCamp/freecodecamp/wiki/Official-Free-Code-Camp-Chat-Rooms and https://github.com/FreeCodeCamp/freecodecamp/wiki/List-of-Free-Code-Camp-city-based-Campsites

### PR DESCRIPTION
**Pull request by abhisekp**
- Add **FCC Official** chat rooms
- Add **FCC City** chat rooms
- Add **camperbot** chat rooms
- Add other miscellaneous chat rooms

Close #148

### CODE REVIEW

**Comment by dcsan** [comment]
did you actually add all the other rooms? i dont see anything new?

**Comment by dcsan** [comment]
rel #148

**Comment by abhisekp** [comment]
@dcsan What other rooms?

File: nap/data/RoomData.js [hunk 1]
```diff
@@ -109,6 +128,85 @@ var bonfireDashedNames = [
     "bonfire-friendly-date-ranges"
 ];
 
+var fccOfficialChatRoomNames = [
```
Comment (abhisekp) [inline nap/data/RoomData.js]: Here is all the FCC official room names.

**Comment by dcsan** [comment]
where are all the local rooms? see the ticket #148

**Comment by abhisekp** [comment]
Adding local rooms

**Comment by abhisekp** [comment]
@dcsan Completed the work.

**Comment by dcsan** [comment]
you could just do a clean checkout, i fixed that wiki issue now.

**Comment by dcsan** [comment]
thanks for this!
will deploy tomorrow / evening just in case problems no time today.

**Comment by abhisekp** [comment]
@dcsan Thanks for giving me an opportunity to contribute.

**Comment by QuincyLarson** [comment]
Note - I've added some campsite rooms that were supposed to be there but weren't. Big thanks to @abhisekp for spotting these!

**Pull request merged**